api/konnect/v1alpha1/konnect_entity_status.go

```diff
@@ -14,6 +14,9 @@ type KonnectEntityStatus struct {
 
 // GetOrgID returns the OrgID field of the KonnectEntityStatus struct.
 func (in *KonnectEntityStatus) GetOrgID() string {
+	if in == nil {
+		return ""
+	}
```

Comment:
Does this work? If `KonnectEntityStatus` is nil, I think that calling this function triggers panic 🤔

Reply:
thanks!